packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -157,18 +150,35 @@ export const buildFilterInputType = (modelType: GraphQLObjectType) => {
   });
 }
 
-export const buildCreateMutationInputType = (modelType: GraphQLObjectType) => {
+// Fetching all fields required for input type including relationships
+const getAllModelFields = (modelType: GraphQLObjectType<any, any, { [key: string]: any; }>, model: ModelDefinition) => {
```

Comment:
Don't need to pass both the graphql type and model definition, since the model definition has both.

Reply:
Not sure what you mean. I'm adding idField here :D